packages/ra-ui-materialui/src/input/BooleanInput.tsx

```diff
@@ -62,7 +63,7 @@ export const BooleanInput = (props: BooleanInputProps) => {
     return (
         <FormGroup
             className={clsx('ra-input', `ra-input-${source}`, className)}
-            {...sanitizeInputRestProps(rest)}
+            row={row}
```

Comment:
`row` and `sx` are really the only props to pass to `FormGroup` component.
And I'm not sure about setting `sx` to both `FormGroup` and `Switch`, so I choose the latter

Reply:
I think you should put the `sx` on the FormGroup. That way, the developer can customize both the FormGroup and the Switch styles